src/pages/createTeam.js

```diff
@@ -8,363 +8,267 @@ import CustomNavbar from "../components/customNavbar.js";
-      onTeam1 = true;
-      teamName1 = tempName;
-      db.collection("users").doc(numHolder).update({
-        onTeam:onTeam1,
-        teamName:teamName1
+const CreateTeam = () => {
+  const [teamName, setTeamName] = useState("");
+  const [redi, setRedi] = useState(false);
+  const [uniq1, setUniq1] = sessionStorage.getItem('uniqname');
+  const [uniq2, setUniq2] = useState("");
+  const [uniq3, setUniq3] = useState("");
+  const [uniq4, setUniq4] = useState("");
+  const [description, setDescription] = useState("");
+  const [show, setShow] = useState(false);
+  const [show2, setShow2] = useState(false);
+  const [uniqname, setUniqname] = sessionStorage.getItem('uniqname');
+  const [usertype, setUserType] = sessionStorage.getItem('user-type');
+  const [onTeam, setOnTeam] = useState(false);
+
+  // set state variables and session storage items
+  useEffect (() => {
+    const getInfo = () => {
+      dbFunctions.getUserInfo(uniqname).then((data) =>{
+        setUserType(data.usertype);
+        setOnTeam(data.usertype === 'team');
+        sessionStorage.setItem('user-type', data.usertype);
+        console.log("user data updated: ", data);
       });
-     break;
-     case 2:
-      numHolder = this.state.uniq2;
-     break;
-     case 3:
-      numHolder = this.state.uniq3;
-     break;
-     case 4:
-      numHolder = this.state.uniq4;
-     break;
-  }
-  dbFunctions.getUserInfo(numHolder).then((data) => {
-    let tempArray = data.invitations;
-    let newVal = data.numInvitations;
-
-    tempArray.push(tempName);
-    if (numHolder !== this.state.uniq1) {
-      db.collection("users").doc(numHolder).update({
-        invitations: tempArray,
-        numInvitations: newVal + 1,
-        onTeam:onTeam1
-    });
-    }
-  })
- }
-
-
- updateInput(event){
-   this.setState({
-     [event.target.name]: event.target.value
-   });
- }
-
-
- addTeam(event){
-  event.preventDefault();
-  const db = firebase.firestore();
-  db.settings({
-    timestampsInSnapshots: true
-  });
-  let tempName = this.state.teamName.split(' ').join('');
-  let uniq4Holder;
-  if (this.state.uniq4 === "") {
-    uniq4Holder = "";
-  }
-  else {
-    uniq4Holder = this.state.uniq4;
-  }
-  dbFunctions.getUserInfo(this.state.uniq1).then(userInfo => {
-    if (userInfo.onTeam === true) {
-      alert(this.state.uniq1+" is already on a team. Please enter a different uniqname");
-      return;
-    }
-    else if (userInfo === "error") {
-      alert(this.state.uniq1+" is not a registered uniqname in the class. Please enter a different uniqname");
-      return;
-    }
-    else {
-
-    }
-    //uniqname2 checks
-    dbFunctions.getUserInfo(this.state.uniq2).then(userInfo1 => {
-      if (userInfo1.onTeam === true) {
-        alert(this.state.uniq2+" is already on a team. Please enter a different uniqname");
+    };
+    getInfo();
+  }, [uniqname]);
+  // rewrote to reduce lines of code
+  const sendData = async(uniq, number, tempName) => {
```

Comment:
rewrote sendData to reduce lines of code by removing unnecessary switch statement and variable numHolder